SU-SWS/saa_victoria_subtheme
# Adapt 664  map link

# READY FOR REVIEW

# Summary
- Adds map link in footer, first column

# Review By (Date)
- 9/4

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. See in footer column one below the address: nice bold map link with external icon, click and it should work

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-664
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
